### ISSUE TITLE
Members are created without usernames / member duplication not possible with more than one member without username

### ISSUE THREAD

**Issue by ColdFireIce** [contao/core#8064]
If you create a member without activating the "Allow login" function, the user will not have a username assigned. The Member list will show a "-" (without the quotes).

Steps:
1. go to account manager/members
2. click new member
3. fill in first name, last name and e-mail address
4. click "save and close"

now the username in the list is: -

![username](https://cloud.githubusercontent.com/assets/5567159/10453926/c6a4c962-71b0-11e5-8f47-27936d3e466d.png)

5. repeat steps 1-4 with other values for e-mail address (2x ?)
6. press the "duplicate member" button for one of these members

This fails with the following error:

```
Fatal error: Uncaught exception Exception with message Query error: Duplicate entry '' for key 'username' (INSERT INTO tl_member (tstamp, firstname, lastname, dateOfBirth, gender, company, street, postal, city, state, country, phone, mobile, fax, email, website, language, groups, login, username, password, assignDir, homeDir, disable, start, stop, dateAdded, lastLogin, currentLogin, loginCount, locked, session, autologin, createdOn, activation, newsletter) VALUES (0, 'test', 'test', '', '', '', '', '', '', '', '', '', '', '', '', '', '', NULL, '', '', '', '', NULL, '', '', '', '1444735247', '0', '0', '3', '0', NULL, NULL, '0', '', '')) thrown in system/modules/core/library/Contao/Database/Statement.php on line 295
#0 system/modules/core/library/Contao/Database/Statement.php(264): Contao\Database\Statement->query()
#1 system/modules/core/drivers/DC_Table.php(968): Contao\Database\Statement->execute()
#2 system/modules/core/classes/Backend.php(650): Contao\DC_Table->copy()
#3 system/modules/core/controllers/BackendMain.php(131): Contao\Backend->getBackendModule('member')
#4 contao/main.php(20): Contao\BackendMain->run()
#5 {main}
```

tested release version: 3.5.4
related issue #8037

**Comment by leofeyer** [comment]
I cannot reproduce this, neither in the online demo nor in my local test environment.

**Comment by leofeyer** [comment]
Ok, I was able to reproduce it thanks to @sascha-mueller's explanation in #8072.

**Comment by leofeyer** [comment]
Fixed in d60458d09dd1ffe5887ab5f07d932f2bca3e4c1e.
